django/core/jinja2/core/member_profiles/retrieve.jinja

```diff
@@ -14,6 +14,23 @@
     ]) }}
 {% endblock %}
 
+{% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="{{ request.build_absolute_uri(absolute_url) }}">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="{{ title }}">
+    <meta property="og:description" content="{{ description }}">
```

Comment:
I think this is a copypasta issue, for MemberProfile this should probably be `name` and `bio` ?